tests/src/Functional/PageHeaderBlockTest.php

```diff
@@ -106,6 +111,23 @@ public function testGuidePageHeaderBlock() {
     $query = $this->xpath('.//h1[contains(concat(" ",normalize-space(@class)," ")," header ")]');
     $found_title = $query[0]->getText();
     $this->assertEquals($found_title, $new_overview_title);
+
+    // Check lede.
+    $this->drupalGet($page->toUrl()->toString());
+    $this->assertSession()->responseContains('Lede to show');
+    // Remove body field, check title and no lede.
+    $field_definitions = \Drupal::service('entity_field.manager')->getFieldDefinitions('node', 'localgov_guides_overview');
+    $field_definitions['body']->delete();
+    $this->drupalGet($overview->toUrl()->toString());
+    $this->assertSession()->responseNotContains('Lede to show');
+    // @todo remove this.
+    drupal_flush_all_caches();
```

Comment:
Test adds checks lede shows in the first place and confirms it works when the field is removed.

However it needs the cache clear, so I'll create a follow-up issue about the caching of the block (guessing it wants cache metadata, rather than just cache tags).